helm/scylla-manager/template.values.schema.json

```diff
@@ -0,0 +1,133 @@
+{
```

Comment:
should be in the other commit

Reply:
I guess I missed that it isn't generated - it should be in the `Add automatic update of CRD schemas in helm charts' schemas` then